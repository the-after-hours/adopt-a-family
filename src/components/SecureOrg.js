// Organization landing page after secure log in

import React from 'react';

class SecureOrg extends React.Component {
  render() {
    return (
      <div>
        <h1>SecureOrg page</h1>
      </div>
    );
  }
}

export default SecureOrg;