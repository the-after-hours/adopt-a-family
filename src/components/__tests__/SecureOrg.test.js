import React from 'react';
import SecureOrg from '../secureOrg.js';
import renderer from 'react-test-renderer';

describe('SecureOrg', () => {
  test('SecureOrg snapshot', () => {
    const component = renderer.create(<SecureOrg />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});