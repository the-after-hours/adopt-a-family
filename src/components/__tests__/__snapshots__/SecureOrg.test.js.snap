// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecureOrg SecureOrg snapshot 1`] = `
<div>
  <h1>
    SecureOrg page
  </h1>
</div>
`;
